<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_fig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Shift figure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_shift_fig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_shift_fig">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_shift_fig">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_fig">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate figure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yz_angle_fig">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_fig">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale figure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_scale_fig">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_scale_fig">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_scale_fig">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xy_angle_fig">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zx_angle_fig">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="import_fig">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import from file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_cam">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add camera</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xy_angle_cam">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_cam">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Shift camera</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zx_angle_cam">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_shift_cam">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yz_angle_cam">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_shift_cam">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_shift_cam">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_cam">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>550</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
